site/src/routes/layout.css

```
@@ -0,0 +1,257 @@
+        }
+    }
+}
+
+html {
+    scroll-behavior: smooth;
+}
+
+body {
+    background-color: var(--color-eve-bg);
+    color: var(--color-eve-text);
+    font-family: "Inter", system-ui, -apple-system, sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)